README.md

```diff
@@ -1,2 +1,40 @@
 # opentelemetry-auto-instr-python
-Auto-Instrumentation for Python (per OTEP 0001)
+The auto-instrumentation for Python (per [OTEP 0001](https://github.com/open-telemetry/oteps/blob/master/text/0001-telemetry-without-manual-instrumentation.md)) instruments each library in a separately installable package to ensure users only need to install the libraries that make sense for their use-case. This repository contains the code initially [donate by DataDog](https://www.datadoghq.com/blog/opentelemetry-instrumentation/) in the `reference` folder. All instrumentation that has been ported lives in the `instrumentors` directory.
+
+# porting ddtrace/contrib to instrumentors
+
+The steps below describe the process to port integrations from the reference directory containing the originally donated code to OpenTelemetry.
+
+1. Move the code into the instrumentors directory
```

Comment:
Is there any reason to use this particular order, I feel that moving the same files twice is not needed.
What about something like:

```
# move code
mkdir -p instrumentors/jinja2/src/opentelemetry/ext/jinja2)
git mv reference/ddtrace/contrib/jinja2/*.py instrumentors/jinja2/src/opentelemetry/ext/jinja2

# move tests 
git mv reference/tests/contrib/jinja2 instrumentors/jinja2/tests

# add readme, setup ....
cp _template/* instrumentors/jinja2/

# move version
mv instrumentors/jinja2/version.py instrumentors/jinja2/src/opentelemetry/ext/jinja2/version.py

```

Reply:
Good point, cleaned up.